directanswercards/card_component.js

```diff
@@ -67,6 +67,9 @@ BaseDirectAnswerCard["{{componentName}}"] = class extends ANSWERS.Component {
         });
       }
     }
+
+    const rtfElement = this._container.querySelector('.js-yxt-rtfValue');
```

Comment:
the SDK does the same thing, but just for my edification are we only adding an event listener to the first instance of .js-yxt-rtfValue?

Reply:
Yes, I believe so. This is how things work in the SDK as well. IIRC, we were told at the time not to worry about multiple RTF fields on a card.